org.eclipse.winery.workflowmodeler/typings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I would recommend to use the new way to install typing definitions (https://blogs.msdn.microsoft.com/typescript/2016/06/15/the-future-of-declaration-files/).
Instead of having a typings.json file you can install your required typings with NPM, e.g. for lodash: `npm install --save-dev @types/lodash`.